File: IQ/Classes/clsCulture.vb
Option Strict On
Option Explicit On
Imports dataAccess

Public Class clsCulture
    Property ID As Integer
    Property Code As String
    Property Name As String
    Public Sub New(ID As Integer, code As String, name As String)
        Me.ID = ID
        Me.Code = code
        Me.Name = name
        iq.Cultures.Add(ID, Me)
        iq.i_culture_code.Add(code, Me)
    End Sub
End Class

File: IQ/Classes/clsCulture.cs
using dataAccess;


public class clsCulture
{
    public int ID { get; set; }
    public string Code { get; set; }
    public string Name { get; set; }
    public clsCulture(int ID, string code, string name)
    {
        this.ID = ID;
        this.Code = code;
        this.Name = name;
        iq.Cultures.Add(ID, this);
        iq.i_culture_code.Add(code, this);
    }
}